src/app/screens/confirmFtTransaction/index.tsx

```diff
@@ -63,6 +65,20 @@ function ConfirmFtTransaction() {
   }, [txError]);
 
   const handleOnConfirmClick = (txs: StacksTransaction[]) => {
+    if (isLedgerAccount(selectedAccount)) {
+      const type: LedgerTransactionType = 'STX';
+      navigate('/confirm-ledger-tx', {
+        state: {
+          unsignedTx: txs[0].serialize(),
+          type,
+          recipients: [{ address: recepientAddress, amountSats: new BigNumber(amount) }],
+          fee: new BigNumber(txs[0].auth.spendingCondition.fee.toString()),
+        },
```

Comment:
can we start typing these state objects which we pass into screens? it should also be validated on the navigated to screen

Reply:
Yeah, currently we type it on the "navigated to" screen https://github.com/secretkeylabs/xverse-web-extension/blob/denys/eng-2499-add-stx-support-for-ledger-accounts/src/app/screens/ledger/confirmLedgerTransaction/index.tsx#L175

Reply:
I just added the types for ledger tx state objects

Reply:
yep, I what I meant was to export the state type and import it from both the 'from' and 'to' screens so we can catch those type bugs directly while coding